fft/Cargo.toml

```diff
@@ -16,8 +16,9 @@ objc = "0.2.7"
 iai = "0.1"
 
 [features]
-metal = ["dep:lambdaworks-gpu"]
-cuda = ["dep:lambdaworks-gpu"]
+default = ["cuda"]
```

Comment:
I think we don't want either by default.

Reply:
Yeah. We used it for local development, and forgot to remove it 😅. Thanks for catching it!